adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -812,6 +812,10 @@ public TokenTask() {
             // Intentionally left blank
         }
 
+        public String getPackageName() {
```

Comment:
This is a bug reported by Findbugs. No one is using this function.  Details of the issue:

 Unread field: com.microsoft.aad.adal.AuthenticationActivity$TokenTask.mPackageName 
 
Bug type URF_UNREAD_FIELD 
In class com.microsoft.aad.adal.AuthenticationActivity$TokenTask
Field com.microsoft.aad.adal.AuthenticationActivity$TokenTask.mPackageName
At AuthenticationActivity.java:[line 819]